io/rootpcm/src/rootclingIO.cxx

```diff
@@ -82,7 +82,9 @@ static bool IsUnsupportedUniquePointer(const char *normName, TDataMember *dm)
          return true;
       }
 
-      clm->BuildRealData();
+      // TODO: Is it not clear what situation we are checking for by checking if
+      // the unique_ptr class has any data members.
+      clm->BuildRealData(nullptr, true);
```

Comment:
I don't think this comment is clear if read out of context from this PR. It also makes me doubt, why are we introducing all this new logic if we don't know which situation it is addressing?

Reply:
This is not new logic.  The comment is pondering why the code has been here for a while.  The change made is to set it to ignore warning and errors related to the schema.